content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Redis Sentinel Ops Requests - Day 2 Lifecycle Management for Redis Sentinel Using KubeDB
+date: 2022-09-29
+weight: 20
+authors:
+- Abdullah Al Shaad
+tags:
+- kubedb
+- Redis
+- sentinel
+- kubernetes
+- high-availability
+- cert-manager
+- opsRequest
+- horizontalScaling
+- enterprise
+- community
+- opsManager
+---
+
+## Summary
+
+On 28th September 2022, Appscode held a webinar on ”Redis Sentinel Ops Requests - Day 2 Lifecycle Management for Redis Sentinel Using KubeDB”. The essential contents of the webinars are:
+- Introducing the concept of Redis Sentinel
+- Deploy and Manage a Cluster in Redis Sentinel Mode
+- Live Demonstration
+- Q & A Session
+
+## Description of the Webinar
+
+Earlier in this webinar, We discussed the `Sentinel` mode in `Redis`. In sentinel mode there are 
```

Comment:
In sentinel mode, there are two types of instances: redis instances and sentinel instances.

Reply:
In sentinel mode, there are two types of Clusters: Redis Database Cluster and Sentinel monitoring Cluster.